controller/revert.go

```diff
@@ -62,20 +60,6 @@ func (c *Controller) Revert(name string) error {
 		return fmt.Errorf("Fail to revert to %v on all replicas", name)
 	}
 
-	if c.Frontend() != "" {
-		if err := c.startFrontend(); err != nil {
-			return err
-		}
-
-		if c.launcher != "" {
-			logrus.Infof("Asking the launcher to start the frontend")
-			if err := c.launcherStartFrontend(); err != nil {
-				if !strings.Contains(err.Error(), "no frontend started since it's not specified") {
-					return err
-				}
-			}
-		}
-	}
```

Comment:
This change is required, without which every single test case related to snapshot revert fails in integration test.
Hence, add the changes similar to the frontend restriction in backup restore.